packages/phoenix-client/src/phoenix/client/resources/experiments/types.py

```diff
@@ -83,7 +90,14 @@ def __post_init__(self) -> None:
 ]
 
 EvaluatorOutput = Union[
-    EvaluationResult, bool, int, float, str, tuple[Score, Label, Explanation], dict[str, Any]
+    EvaluationResult,
+    bool,
+    int,
+    float,
+    str,
+    tuple[Score, Label, Explanation],
+    dict[str, Any],
+    "ScoreResult",
```

Comment:
Again feels like we really are making our jobs harder by adding to the union type

Reply:
I also disagree here, by convention we allow users to pass in raw as opposed to a decoracted function. If we natively support our own evals, then a callable that returns scores should be accepted